openpype/hosts/resolve/api/lib.py

```diff
@@ -89,11 +89,17 @@ def get_current_project():
     """Get current project object.
     """
     if not self.current_project:
-        self.current_project = get_project_manager().GetCurrentProject()
+        set_current_project()
 
     return self.current_project
```

Comment:
Doesn't it just make sense to just never cache `current_project`?

Isn't it so that you can technically close a project and open a new one in the same resolve session meaning `self.current_project` would be inaccurate from that moment forward anyway?